genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/JobDirectoryManifest.java

```diff
@@ -223,11 +222,13 @@ public long getTotalSizeOfFiles() {
 
         private final Path root;
         private final ImmutableMap.Builder<String, ManifestEntry> builder;
+        private final Metadata metadata;
```

Comment:
The commit message doesn't seem to match what is in the commit?

Reply:
Ah indeed!